Makefile

```diff
@@ -258,3 +266,24 @@ kind-load-images:
             $(REGISTRY)/test-backend:dev \
             $(REGISTRY)/jmalloc/echo-server:dev \
             $(REGISTRY)/busybox:dev
+
+
+GATEWAY_API_PACKAGE ?= sigs.k8s.io/gateway-api
+GATEWAY_API_VERSION ?= v0.5.1
+GATEWAY_API_CRDS_LOCAL_PATH = $(shell go env GOPATH)/pkg/mod/$(GATEWAY_API_PACKAGE)@$(GATEWAY_API_VERSION)/config/crd
+
+.PHONY: go-mod-download-gateway-api
+go-mod-download-gateway-api:
+	@go mod download $(GATEWAY_API_PACKAGE)
+
+### install:				Install Gateway API CRDs into the K8s cluster.
+.PHONY: install-gateway-api
+install-gateway-api: go-mod-download-gateway-api
+	kubectl apply -k $(GATEWAY_API_CRDS_LOCAL_PATH)
+	kubectl apply -k $(GATEWAY_API_CRDS_LOCAL_PATH)/experimental
+
+### uninstall-gateway-api:	Uninstall Gateway API CRDs from the K8s cluster.
+.PHONY: uninstall-gateway-api
+uninstall-gateway-api: go-mod-download-gateway-api
```

Comment:
why we need go-mod-download-gateway-api?

Reply:
Dependent updates are convenient.

Reply:
but its uninstall-gateway-api

Reply:
Isn't it weird that I'm being asked to update it first when I want to delete it?

Reply:
This package does not exist when uninstalling is mainly avoided.

Reply:
This is obviously not necessary.

I can get the name of the CRD in the following two ways

```
kubectl api-resources --api-group=gateway.networking.k8s.io -o name
```

or

```
kubectl get crds  | awk '/gateway.networking.k8s.io/ {print $1}'
```

Then I can get the output:

```
gatewayclasses.gateway.networking.k8s.io
httproutes.gateway.networking.k8s.io
referencepolicies.gateway.networking.k8s.io
tcproutes.gateway.networking.k8s.io
tlsroutes.gateway.networking.k8s.io
udproutes.gateway.networking.k8s.io
```

Just need a `for` loop, you can delete these CRD resources directly.

This does not require any dependencies, even an offline environment.

Reply:
This is not very convenient. There is also a gateway-api webhook.